src/test/java/redis/clients/jedis/tests/commands/ClusterBinaryJedisCommandsTest.java

```diff
@@ -186,14 +189,18 @@ private static String getNodeId(String infoOutput) {
   }
 
   private void waitForClusterReady() throws InterruptedException {
-    boolean clusterOk = false;
-    while (!clusterOk) {
```

Comment:
This loop used to check a condition after each 50ms. Don't we need repeated checking anymore?